topology/mask.h

```diff
@@ -444,7 +444,7 @@ class DifferenceMask : public Mask< D >
   Mask< D >* clone() const;
 
 protected:
-  Mask< D > *mask1_, *mask2_;
+  Mask< D >* mask1_, *mask2_;
 };
 
 
```

Comment:
This file has nothing to do with the pull request and just got in due to clang reformatting. Please remove from PR.